pkg/defaults/policies/files/host_pids.json

```diff
@@ -11,13 +11,75 @@
     "DEPLOY"
   ],
   "exclusions": [
+    {
+      "name": "Don't alert on deployment apiserver-watcher-* in kube-system namespace",
+      "deployment": {
+        "name": "apiserver-watcher-.*",
+        "scope": {
+          "namespace": "kube-system"
+        }
+      }
+    },
+    {
+      "name": "Don't alert on deployment apiserver-watcher-* in openshift-kube-apiserver namespace",
```

Comment:
This is redundant because we have the exclusion below that excludes the entire namespace anyway